profiles/package.mask

```diff
@@ -366,56 +358,6 @@ app-admin/super-user-spark
 # to date. Masked for removal.
 sci-mathematics/agda-lib-ffi
 
-# Jack Todaro <solpeth@posteo.org> (2020-12-16)
-# Too many packages require <dev-haskell/aeson-1.5.
-# The following package versions are masked until
-# aeson-1.5 is possible:
->=dev-haskell/aeson-1.5
->=dev-haskell/free-5.1.4
->=dev-haskell/random-1.2
->=dev-haskell/splitmix-0.1
->=dev-haskell/strict-0.4
->=dev-haskell/th-abstraction-0.4
->=dev-haskell/these-1.1
->=dev-haskell/quickcheck-instances-0.3.23
->=dev-haskell/recursion-schemes-5.2.1
->=dev-haskell/generic-deriving-1.14
->=dev-haskell/deriving-compat-0.5.10
->=dev-haskell/bifunctors-5.5.8
->=dev-haskell/invariant-0.5.4
->=dev-haskell/microlens-th-0.4.3.8
->=dev-haskell/bifunctors-5.5.9
->=dev-haskell/profunctors-5.6.1
->=dev-haskell/rebase-1.10.0.1
->=dev-haskell/semigroups-0.19.1
->=dev-haskell/dlist-1.0
->=dev-haskell/selective-0.4.1.1
->=dev-haskell/byline-1.1.0.1
->=dev-haskell/optparse-applicative-0.16.1.0
->=dev-haskell/vimeta-0.3.0.1
->=dev-haskell/persistent-2.11.0.2
->=dev-haskell/persistent-template-2.9.1.0
->=dev-haskell/web-plugins-0.4
->=dev-haskell/clckwrks-0.26.3
->=dev-haskell/clckwrks-plugin-page-0.4.3.24
->=dev-haskell/hashable-1.3.0.0
->=dev-haskell/binary-tagged-0.3
->=dev-haskell/unordered-containers-0.2.13.0
->=dev-haskell/binary-instances-1.0.1
->=dev-haskell/scotty-0.12
-
-# Jack Todaro <solpeth@posteo.org> (2020-08-12)
-# Many packages require QuickCheck < 2.14. Mask
-# QuickCheck >= 2.14 and its reverse dependencies
->=dev-haskell/quickcheck-2.14
->=dev-haskell/hspec-discover-2.7.2
->=dev-haskell/hspec-core-2.7.2
->=dev-haskell/hspec-2.7.2
-
```

Comment:
The above unmasks do not seem to relate to the stack bump. Could you please explain this one to me?

Reply:
Hm, these must have been why I thought I needed #1118...I'll need to recreate things, see why I thought this was needed.